subgraphs/curve/deployment/deployment.js

```diff
@@ -0,0 +1,104 @@
+import * as protocolNetworkData from './deploymentConfigurations.json' assert {type: "json"}
```

Comment:
why are you duplicating these deployment files per subgraph? can't they be shared?